core/src/main/java/nl/nn/adapterframework/http/rest/ApiServiceDispatcher.java

```diff
@@ -179,7 +179,7 @@ protected JsonObject generateOpenApiJsonSchema(Collection<ApiDispatchConfig> cli
 					if (StringUtils.isNotEmpty(adapter.getDescription())) {
 						methodBuilder.add("summary", adapter.getDescription());
 					}
-					methodBuilder.add("operationId", adapter.getName());
+					methodBuilder.add("operationId", listener.getName());
```

Comment:
I do agree that the operationId should be based on receiver or listener, and not on adapter, but the name attribute as it is right now is not a nice solution, because it tends to get a name like ``listener of [Json2XmlValidator-API-POST]``.
Will this be solved when you extend this exit.elementName solution to the input? Maybe introduce a 'operationId' attribute in ApiListener then.